README.md

```diff
@@ -1,2 +1,35 @@
 # action-af
-A GitHub Action for running ZAP Automation Framework plans.
\ No newline at end of file
+A GitHub Action for running [ZAP Automation Framework](https://www.zaproxy.org/docs/automate/automation-framework/) plans.
+
+## Inputs
+
+### `plan`
+
+**Required** The file system path to the Automation Framework plan to run.
+
+### `docker_name`
+
+**Required** The name of the [ZAP Docker image](https://www.zaproxy.org/docs/docker/about/#install-instructions) to be used. By default the action runs the stable image.
```

Comment:
This is confusing, it's required but there's a default?

Reply:
If there's a default why require setting it... (make it optional if they want something other than default)

Reply:
You don't need to specify it but if you do it must not be empty, that's why the required. Would "mandatory" be better?

Reply:
Optional, if specified must not be empty? Or if specified but empty the default (stable) is used...

Something like that. Maximize user friendliness 👍

Reply:
Changes pushed.